### PR TITLE
Setup Travis CI for AWS Elastic Beanstalk

### PR DESCRIPTION
**Pull request by rhernand3z**
Updated documentation and travis.yml -- ready for production

### CODE REVIEW

**Comment by rhernand3z** [comment]
reminding you about this PR, getting ready to showcase transferred apps working with Travis on our AWS stack.. 

CC: @kfriedman @EdwinGuzman @ktp242

File: ReadMe.md [hunk 1]
```diff
@@ -3,31 +3,47 @@
 > 1.3.9
 
 ## Installation
-Install all dependencies listed under `package.json`
+Install all NPM dependencies listed under `package.json`
```
Comment (EdwinGuzman) [inline ReadMe.md]: Lowercase`npm`.

File: ReadMe.md [hunk 2]
```diff
@@ -3,31 +3,47 @@
...
+$ APP_ENV=production NODE_ENV=production npm start
 ```
 
+## GIT Workflow
+We follow a [feature-branch](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) workflow. If you need to introduce/update the application code, you `SHOULD`:
+
+* Create a new branch off the `development` branch
+* Send a PR pointing to the `development` branch upon completion
+* Once the PR is approved, it should be merged into the `development` branch
+* If there are several PR's in process, a release should be scheduled by merging all completed PR's into the `development` branch
```
Comment (ktp242) [inline ReadMe.md]: PRs? If in plural?

**Comment by holingpoon** [comment]
Taking over this PR :smile: 

The plan is to upgrade `.travis.yml` so it will support `development`, `qa`, and `master` branches for Travis CI AWS integration. Stay tuned.

File: .travis.yml [hunk 3]
```diff
@@ -1,24 +1,52 @@
 language: node_js
 node_js:
-  - "8"
+- '8'
```
Comment (EdwinGuzman) [inline .travis.yml]: I think this should be indented, no?

File: .travis.yml [hunk 4]
```diff
@@ -1,24 +1,52 @@
...
 cache:
   directories:
-    - node_modules
+  - node_modules
```
Comment (EdwinGuzman) [inline .travis.yml]: This one as well.

File: ReadMe.md (hunk 1 above)
Comment (EdwinGuzman) [reply ReadMe.md]: @holingpoon can you update this too?

File: ReadMe.md (hunk 2 above)
Comment (EdwinGuzman) [reply ReadMe.md]: Hmm yeah, plural. He meant `PRs` and not `PR's`.

**Comment by holingpoon** [comment]
@EdwinGuzman, @ktp242 Please re-review at your leisure. Thanks!

File: ReadMe.md [hunk 5]
```diff
@@ -2,32 +2,54 @@
 ## Version
 > 1.3.9
 
+| Branch         | Status|  
+|:---------------|:---------------------------------------------------------------------------------------------------------------------------|  
+| `production`   | [![Build Status](https://travis-ci.org/NYPL/dgx-homepage.svg?branch=master)](https://travis-ci.org/NYPL/dgx-homepage)      |  
+| `qa`           | [![Build Status](https://travis-ci.org/NYPL/dgx-homepage.svg?branch=development)](https://travis-ci.org/NYPL/dgx-homepage) |
```
Comment (ktp242) [inline ReadMe.md]: is this should be `[Build Status]`

and the next one should be 
[Build Status]

File: ReadMe.md (hunk 1 above)
Comment (holingpoon) [reply ReadMe.md]: @EdwinGuzman Yup doing it

File: README.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+* All releases merged into `master` `MUST` be tagged and pushed to Github with their corresponding `version`
+
+## Components
+The homepage is made out of different React components that are displayed as features separated into rows. The React components are separate repos, so the homepage acts as a container that renders those components.
+
+Going from the top to the bottom of the page, we have the following list of components. All the components, **except** for the Content Banner component, are wrapped in the [dgx-homepage-row-component](git+ssh://git@bitbucket.org/NYPL/dgx-homepage-row-component.git) component. It is a container to render the feature for each row along with a left sidebar. Since the Content Banner takes up the whole width of the page, it is not wrapped in the dgx-homepage-row-component container.
+
+| Component | Repo |
+|---|---|
+| Content Banner | [dgx-homepage-content-banner](git+ssh://git@bitbucket.org/NYPL/dgx-homepage-content-banner) |
```
Comment (EdwinGuzman) [inline README.md]: Hmm can you remove the `git+ssh` in the urls here and just make it point to bitbucket? It doesn't work otherwise.

**Comment by EdwinGuzman** [comment]
Just minor change and fix the conflicts.

**Pull request merged**